tikv-client/src/main/java/com/pingcap/tikv/operation/iterator/CoprocessorIterator.java

```diff
@@ -83,7 +83,8 @@
         session,
         SchemaInfer.create(dagRequest),
         dagRequest.getPushDownType(),
-        dagRequest.getStoreType()) {
+        dagRequest.getStoreType(),
+        dagRequest.getStartTs().getVersion()) {
```

Comment:
question: can we always get the startts from dagrequest?

Reply:
always.